README.md

```diff
@@ -21,24 +28,67 @@ A curated list of IPv6 materials and learning resources
 - [Internet Protocol Version 6 (IPv6) Basics Cheat Sheet by Jens Roesen](https://www.roesen.org/files/ipv6_cheat_sheet.pdf)
 - [IPv6 Addressing Guide (pt-BR)](http://ipv6.br/media/arquivo/ipv6/file/46/enderec-v6.pdf)
 
-
 ## Courses
 - [IPv6 Training Recommendations from Internet Society](https://www.internetsociety.org/deploy360/ipv6/training/)
 - [Linux IPv6 Crash Course](https://www.linux.com/tutorials/ipv6-crash-course-linux/)
 
-## Tools and Commands
-- [`ping6`](https://linux.die.net/man/8/ping6)
-  - `ping6 <node address>` - Ping a node
-  - `ping6 ff02::1%<interface>` - Ping all-nodes multicast address on link
+## Tools
+- [Bash ULA Generator](https://github.com/adeverteuil/bash-ula-generator)
+- [Radvd](https://github.com/reubenhwk/radvd)
+
+## Commands
```

Comment:
How about just putting everything under Tools and getting rid of Commands? I think it makes things clearer.

Reply:
I am not attached to commands, even though I think Commands makes more sense. But I moved it to tools, since it is fine for me :)